### PR TITLE
graph-tool 2.43

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

File: Formula/graph-tool.rb [hunk 1]
```diff
@@ -32,29 +32,31 @@ class GraphTool < Formula
   depends_on "librsvg"
   depends_on macos: :mojave # for C++17
   depends_on "numpy"
+  depends_on "pillow"
   depends_on "py3cairo"
   depends_on "pygobject3"
   depends_on "python@3.9"
   depends_on "scipy"
+  depends_on "six"
```
Comment (SMillerDev) [inline Formula/graph-tool.rb:40]: Are these two requirements new?

Comment (nandahkrishna) [reply Formula/graph-tool.rb:40]: `six` has been removed as a resource and made a dependency, I'm unsure about `pillow`.

Comment (chenrui333) [reply Formula/graph-tool.rb:40]: pillow was listed when I did the python dependencies check, let me see if i can run without it.

Comment (chenrui333) [reply Formula/graph-tool.rb]: > `six` has been removed as a resource and made a dependency, I'm unsure about `pillow`.

this is the common practice we are doing right now.

**Review by nandahkrishna — APPROVED** [review]

File: Formula/graph-tool.rb (hunk 1 above)
Comment (carlocab) [reply Formula/graph-tool.rb]: Installing `graph-tool` using `conda` installs `pillow` though, so it might be needed at runtime at some point

**Pull request closed without merging**